src/grz_cli/models/config.py

```diff
@@ -93,10 +103,33 @@ class S3Options(StrictBaseModel):
     """
 
 
-FilePath = Annotated[Path, AfterValidator(lambda v: v.expanduser()), PathType("file")]
+class PruefberichtConfig(StrictBaseModel):
+    authorization_url: Annotated[AnyHttpUrl, UrlConstraints(allowed_schemes=["https"], host_required=True)] | None = (
```

Comment:
Can probably shorten this to `auth_url`.

Reply:
Or we can just leave it like this ;)